src/pages/ReimbursementAccount/ReimbursementAccountForm.js

```diff
@@ -40,8 +40,9 @@ class ReimbursementAccountForm extends React.Component {
     }
 
     render() {
-        const isErrorVisible = _.size(lodashGet(this.props, 'reimbursementAccount.errors', {})) > 0
-            || lodashGet(this.props, 'reimbursementAccount.error', '').length > 0;
+        const hasErrorFields = _.size(this.props.reimbursementAccount.errorFields) > 0;
+        const error = _.last(_.values(this.props.reimbursementAccount.errors));
```

Comment:
We can use [getLatestErrorMessage](https://github.com/Expensify/App/blob/ce1ceac6f1563d4113af60c63e88f383dc64305d/src/libs/ErrorUtils.js#L39-L52) here